src/libs/ReportUtils.js

```diff
@@ -1785,16 +1785,31 @@ function getTransactionReportName(reportAction) {
     });
 }
 
+/**
+ * Get actor name to display in the message preview
+ *
+ * @param {Object} report
+ * @param {Number} actorID
+ * @param {Boolean} shouldShowWorkspaceName
+ * @param {Boolean} shouldUseShortForm
```

Comment:
Looks like these are optional arguments.

```suggestion
 * @param {Boolean} [shouldShowWorkspaceName]
 * @param {Boolean} [shouldUseShortForm]
```